File: SGA_Api/SGA_Api/Models/Traspasos/CrearTraspasoDto.cs
namespace SGA_Api.Models.Traspasos
{
	public class CrearTraspasoDto
	{
		public string AlmacenOrigen { get; set; }
		public string UbicacionOrigen { get; set; }
		public Guid PaletId { get; set; }
		public int UsuarioInicioId { get; set; }
		public string CodigoPalet { get; set; }
	}
}

File: SGA_Android/app/src/main/java/com/example/sga/data/dto/traspasos/CrearTraspasoDto.kt
package com.example.sga.data.dto.traspasos

data class CrearTraspasoDto(
    val almacenOrigen: String,
    val ubicacionOrigen: String,
    val paletId: String,
    val usuarioInicioId: Int,
    val codigoPalet: String
) 